docs/DeveloperGuide.adoc

```diff
@@ -723,15 +723,71 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
+|`* * *` |new user |import existing contacts |add contacts quickly when starting to use the App or reinstalling the App
+
+|`* * *` |frequent user |backup existing contacts as I add/remove them |restore a previous version of my contacts if I make a big mistake
```

Comment:
Here the reason can probably be just able to back up existing contacts (`as I add/remove them` is a bit impossible cause then you will have MANY versions) unless you're planning to do a RCS like Git :P

reason can just be prevent file corruption or etc. Undo already exist in current ab4 😄